source/tit/py/module.cpp

```
@@ -38,10 +41,34 @@
 
+void Module::add(CStrView name, const Object& obj) const {
+  ensure(PyModule_AddObjectRef(get(), name.c_str(), obj.get()));
+}
+
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 auto import_(CStrView name) -> Module {
   return steal<Module>(ensure(PyImport_ImportModule(name.c_str())));
 }
 
+auto module_(std::string name) -> Module {
```

cppcheck (2.10-2)

[style] The function 'module_' is never used.

(unusedFunction)